src/scanner.c

```diff
@@ -102,7 +102,10 @@ typedef enum Token Token;
 /*
  * Helpful macros
  */
-#ifdef TREE_SITTER_INTERNAL_BUILD
+#ifdef __wasm
+#define DEBUG(...)
```

Comment:
With the updated tree sitter, these debug calls should be replaced with calls to `lexer->log(...)`

Reply:
Yes, this PR also need a test, I was so happy it worked that just sent it as is 😅